Update README.md (#196)
Adyen/adyen-python-api-library: README.md

~~~~ python
import Adyen

adyen = Adyen.Adyen()

adyen.payment.client.xapikey = "YourXapikey"
adyen.payment.client.hmac = "HMAC key for skin code"
adyen.payment.client.platform = "test" # Environment to use the library in.
adyen.payment.client.merchant_account = "merchant account name from CA"
~~~~
### Consuming Services
Every API the library supports is represented by a service object. The name of the service matching the corresponding API is listed in the [Integrations](#supported-api-versions) section of this document.
#### Using all services
~~~~python
import Adyen
adyen = Adyen.Adyen()

> adyen.payment.client.xapikey = "YourXapikey"

Reviewer: ```suggestion
adyen.payment.client.xapikey = "YOUR_XAPI_KEY"
```
This might be a little bit more consistent

adyen.payment.client.platform = "test"  # change to live for production
request = {
      "amount": {
        "currency": "USD",
        "value": 1000 # value in minor units
      },
      "reference": "Your order number",
      "paymentMethod": {
        "type": "visa",
        "encryptedCardNumber": "test_4111111111111111",
        "encryptedExpiryMonth": "test_03",
        "encryptedExpiryYear": "test_2030",
        "encryptedSecurityCode": "test_737"
      },
      "shopperReference": "YOUR_UNIQUE_SHOPPER_ID_IOfW3k9G2PvXFu2j",
      "returnUrl": "https://your-company.com/...",
      "merchantAccount": "YOUR_MERCHANT_ACCOUNT"
    }
result = adyen.checkout.payments(request)
~~~~
#### Using one of the services
~~~~python
from Adyen import AdyenCheckoutApi
adyen = AdyenCheckoutApi()

> adyen.client.xapikey = "YourXapikey"

Reviewer: ```suggestion
adyen.payment.client.xapikey = "YOUR_XAPI_KEY"
```
Actually, also full caps looks a bit ugly so your call I guess.

~~~~
pass the dictionary to the method as an additional argument.
~~~~ python
adyen.management.account_company_level_api.get_companies(query_parameters=query_parameters)
~~~~
### Handling exceptions

> Adyen service exceptions are extending the AdyenError class. After you catch this exception, you can access the 

Reviewer: ```suggestion
Adyen service exceptions extend the AdyenError class. After you catch this exception, you can access the 
```
